src/passes/Print.cpp

```diff
@@ -2464,9 +2475,9 @@ void PrintSExpression::printFullLine(Expression* expression) {
     doIndent(o, indent);
   }
   if (full) {
-    o << "[";
+    o << "(; ";
```

Comment:
While we're at it, can we move the type comment to the end of the line rather than the beginning? That way the instructions will still line up nicely on the left side.

Reply:
Done.